ProjectEureka/project-eureka-flutter
# PE-38 | Implemented General and Payment pages

WHAT I DID:

- Implemented General settings page. It has 3 setting switches and an About button that shows some information about Eureka.
- Implemented Payment settings page. It has 2 options: Google pay and Apple pay. Pages redirect users to the blank pages in which specific settings will be implemented in the next task. Blank pages have confirmation buttons that represent successfully saved data in the application.

HOW TO TEST

- Make sure to `flutter pub get`
- Settings must initially be switched off, but once changed, it must store persistent data and show the last changes after leaving the page or after closing the application.
![Screenshot_3](https://user-images.githubusercontent.com/68677129/107596873-f2265c00-6bcd-11eb-997e-6b147870cc1b.jpg)
- About page must appear on top of the settings page without overflowing and being able to close.
- Google Pay and Apple pay have logos and those must be pulled from Assets without errors or delays.
![Screenshot_4](https://user-images.githubusercontent.com/68677129/107596936-18e49280-6bce-11eb-9443-1b464656acb4.jpg)
- Confirmation message must redirect user back to settings without being able to go back to Google Pay or Apple Pay after confirmation alert.
